examples/KelvinHelmholtz/include/simulation_defines/param/radiationConfig.param

```diff
@@ -111,11 +111,6 @@ namespace radiationNyquist
 
 namespace parameters
 {
-/*!enable radiation calculation*/
-#ifndef PARAM_RADIATION
-#define PARAM_RADIATION 0
-#endif
-#define ENABLE_RADIATION PARAM_RADIATION
```

Comment:
REQUIRED or OPTIONAL:
The Kelvin Helmholtz example includes a radiation test 
```c++
flags[2]="-DCUDA_ARCH=35 -DPARAM_OVERWRITES:LIST=-DPARAM_RADIATION=1"
```

Please: either remove this test **or** add such a guard for the attribute `momentumPrev1`